blog/_posts/2017-11-01-zinc-blog-1.0.md

```diff
@@ -0,0 +1,258 @@
+lines of code and such complexity. Why was that the case?
+  
+Zinc 0.13.x has always had an underlying dependency detection operating at the
+source rather than the class level. In practice, this means that the
+invalidation algorithm collected more recompilation candidates than it should:
+all classes defined in a source file that were "affected" by a change would be
+checked for invalidation *transitively*. As it is common to define more than one
+class per source file, the number of source files to recompile grew up quickly
+and resulted in worse incremental compiles.
+  
+However, Zinc 1.x changes the dependency detection to operate at the class
+level. With the new detection, the incremental compiler has a mapping of source
+files with their defined classes, class names with their generated class files
+and class names with their dependent class names. This information, complemented
+with other mappings, allows the incremental compiler to reason about class
+dependencies directly and therefore be much more selective when picking
+candidates for invalidation.
+  
+In general, Zinc 1.0 is faster and more correct than 0.13.x (thanks to some
+fundamental improvements to the Scala bridges implementation). If you'd like to
+know more about the inner workings of the Scala incremental compiler, [subscribe
+to the following ticket](https://github.com/sbt/zinc/issues/435); at some point
+in the next months, I'd like to write up a technical document for future
+contributors.
+  
+### Try it out!
+  
+Zinc is already available in [sbt 1.0](https://github.com/sbt/sbt), and [it will
+soon be available in Pants](https://github.com/pantsbuild/pants/pull/4729). If
+you're **an sbt user**, I recommend you to upgrade to latest sbt 1.0.3 and give
+it a spin!
+  
+If you're a library author, add [Zinc
+1.0.3](http://search.maven.org/#artifactdetails%7Corg.scala-sbt%7Czinc_2.11%7C1.0.3%7Cjar)
+to your `build.sbt` with `libraryDependencies += "org.scala-sbt" % "zinc_2.11" %
+"1.0.3"`. To add it to other builds tools like Maven, click
+[here](http://search.maven.org/#artifactdetails%7Corg.scala-sbt%7Czinc_2.11%7C1.0.3%7Cjar).
+  
+### Special acknowledgements
+  
+I'd like to thank the sbt team, and concretely Eugene Yokota, for reviewing all
+our work and always be open for improvements. I'm happy to see such cross-team
+collaboration work in a real-world project like Zinc.
+  
+### Conclusion
+
+With this announcement, I explain the work that the Scala Center has put
+into the already released Zinc 1.0 to motivate the work carried out these previous months.
+  
+While happy with the results, I still believe that there's more room for
+improvement, [especially when it comes to faster incremental compiles and
+reproducibility](https://github.com/scalacenter/advisoryboard/blob/master/proposals/015-zinc-performance.md).
+In the next months, we will zoom on in these problems to continue improving the
```

Comment:
Should be "zoom in on these problems", no?

Reply:
Yes